core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/internal/signer/DefaultAwsV4HttpSigner.java

```diff
@@ -112,6 +122,14 @@ private static boolean isEventStreaming(SdkHttpRequest request) {
         return "application/vnd.amazon.eventstream".equals(request.firstMatchingHeader("Content-Type").orElse(""));
     }
 
+    private static boolean isChunkedEncoding(SdkHttpRequest request) {
+        return "aws-chunked".equals(request.firstMatchingHeader(Header.TRANSFER_ENCODING).orElse(""));
+    }
```

Comment:
It is still unclear to me where this should get set (does signed do it or not? If not, we need another mechanism for determining chunked-encoding or not). We should iron this out.